samples/azure_attestation/ra_client/client.go

```diff
@@ -87,60 +61,28 @@ func main() {
 	fmt.Println("🔒 Sent secret over attested TLS channel.")
 }
 
-// verfiyTokenClaims verifies a bunch of important token claims.
-// For productive envornments, please refere to the MS Azure Attestation Documentation
-// and JWT as well as OpenID Connect specification to determine which parameters need
-// to be verified.
-func verifyTokenClaims(publicClaims jwt.Claims, privateClaims privateClaims, signer string) error {
-	if err := publicClaims.Validate(jwt.Expected{Issuer: issAddr, Time: time.Now()}); err != nil {
-		return fmt.Errorf("issuer of token could not be verified: %v", err)
-	}
-	fmt.Printf("✅ Issuer of token is %s.\n", publicClaims.Issuer)
-
-	if privateClaims.SignerID != signer {
+// verifyReportValues compares the report values with that were defined in the
+// enclave.json and that were included into the binary of the server during build.
+func verifyReportValues(report attestation.Report, signer string) error {
+	if !reflect.DeepEqual(report.SignerID, []byte(signer)) {
```

Comment:
You also need to change the sample:
```suggestion
func verifyReportValues(report attestation.Report, signer []byte) error {
	if !bytes.Equal(report.SignerID, signer) {
```
pass decoded signer (like https://github.com/edgelesssys/ego/blob/master/samples/remote_attestation/ra_client/client.go#L24)